architecture/README.md

```diff
@@ -12,148 +11,149 @@ types of roles. The TiDB community has the following organizations:
     * [Active Contributor](#active-contributor)
     * [Reviewer](#reviewer)
-        * [How to become a Reviewer?](#how-to-become-a-reviewer)
-        * [Responsibilities and Privileges](#responsibilities-and-privileges)
+        * [How to become a Reviewer](#how-to-become-a-reviewer)
+        * [Privileges and responsibilities](#responsibilities-and-privileges)
     * [Committer](#committer)
-        * [How to become a Committer?](#how-to-become-a-committer)
-        * [Responsibilities and Privileges](#responsibilities-and-privileges-1)
-        * [Definition of Inactive Committer and operations:](#definition-of-inactive-committer-and-operations)
-        * [Committer List](#committer-list)
+        * [How to become a Committer](#how-to-become-a-committer)
+        * [Privileges and responsibilities](#responsibilities-and-privileges-1)
     * [Maintainer](#maintainer)
-        * [How to become a Maintainer?](#how-to-become-a-maintainer)
-        * [Responsibilities and Privileges](#responsibilities-and-privileges-2)
+        * [How to become a Maintainer](#how-to-become-a-maintainer)
+        * [Privileges and responsibilities](#responsibilities-and-privileges-2)
 * [TiDB User Group](#tidb-user-group)
 * [Organizer Committee](#organizer-committee)
 
 <!-- vim-markdown-toc -->
 
 ## PMC
 
-TBD
+PMC (Project Management Committee) functions as the core management team that oversees the TiDB community. See [Governance](../GOVERNANCE.md) for more details about PMC and [PMC List](./pmc-list.md) for a list of incumbent PMC members.
 
 ## TiDB Developer Group
 
-### Contributor
+As the cornerstone of community development, the TiDB Developer Group consists of these roles: Maintainer, Committer, Reviewer, Active Contributor, and Contributor. Each role takes corresponding responsibilities in the community. Collectively they play an important role in the robust development of TiDB.
 
-A Contributor refers to the person who contributes to the following projects:
+### Contributor
 
-* TiDB: https://github.com/pingcap/tidb
-* Parser: https://github.com/pingcap/parser
-* TiKV: https://github.com/tikv/tikv
-* TiSpark: https://github.com/pingcap/tispark
-* PD: https://github.com/pingcap/pd
-* Docs: https://github.com/pingcap/docs
-* Docs-cn: https://github.com/pingcap/docs-cn
+Anyone who has one PR merged for any project in the [PingCAP Organization](https://github.com/pingcap) is a Contributor. New Contributors should be welcomed to the community by existing members, helped with PR workflow, and directed to relevant documentation and communication channels. A contributor becomes a member of the SIG of the contribution automatically.
 
-#### How to become a Contributor?
+#### How to become a Contributor
 
-If a PR (Pull Request) submitted to any one of the above projects by you is
-approved and merged, then you become a Contributor.
+- merged at least 1 PR for any project in the [PingCAP Organization](https://github.com/pingcap)
```

Comment:
```suggestion
- At least 1 PR being merged for any project in the [PingCAP Organization](https://github.com/pingcap)
```

Reply:
"merged" here is a consistent usage with other requirements. Here the full sentence is intended as "(You must) have...